cmd/stellar-rpc/internal/daemon/daemon.go

```diff
@@ -235,6 +250,17 @@ func createStellarCoreClient(cfg *config.Config) stellarcore.Client {
 	}
 }
 
+func createHighperfStellarCoreClient(cfg *config.Config) interfaces.FastCoreClient {
+	// It doesn't make sense to create a client if the local server is not enabled
+	if cfg.CaptiveCoreHTTPQueryPort == 0 {
+		return nil
+	}
+	return &stellarcore.Client{
+		URL:  fmt.Sprintf("http://localhost:%d", cfg.CaptiveCoreHTTPQueryPort),
```

Comment:
This should use `cfg.StellarCoreURL` over localhost, right?

Reply:
Not really. Because:
1. The url includes the port
2. The user may choose a different host for the normal (non-querry) http requests